RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -630,7 +630,7 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
     <value>Wählen Sie einen Parameter und benutzen Sie die Schaltflächen oder doppelklicken Sie um ihn zu bewegen oder zu löschen</value>
   </data>
   <data name="Restore" xml:space="preserve">
-    <value>Wiederherstellen</value>
+    <value>Wieder-herstellen</value>
```

Comment:
I took a while until I decided to use the hyphen here.
Currently this string is only used once. 
Without the hyphen the text is shown in this way in the button:
Wiederhers
tellen
That is the reason for the hyphen.
If you know how a conditional hypen can be used. Please let me know.